rewrites.js

```diff
@@ -176,6 +176,11 @@ exports.REWRITES = [
     source: '/:collectiveSlug/:verb(tiers|contribute|events|projects|connected-collectives)',
     destination: '/contribute',
   },
+  {
+    source:
+      '/:parentCollectiveSlug?/:collectiveType(events|projects)?/:collectiveSlug/:verb(tiers|contribute|events|projects|connected-collectives)',
+    destination: '/contribute',
+  },
```

Comment:
Does it create conflicts if we merge these? Since the first parameters are optional, it sounds like it would have the same effect

```suggestion
      '/:parentCollectiveSlug?/:collectiveType(events|projects)?/:collectiveSlug/:verb(tiers|contribute|events|projects|connected-collectives)',
    destination: '/contribute',
  },
```